### DISCUSSION TITLE
wan6口没ipv6客户端解决方案

### DISCUSSION THREAD

**Discussion by smomop** [coolsnowwolf/lede#9557] [R68S / 电犀牛专区]
软件包安装红框软件
![Snipaste_2022-06-13_10-16-11](https://user-images.githubusercontent.com/63437136/173267729-a45280de-a926-4b3d-b85a-7b317645e5a8.jpg)
<img width="1331" alt="image" src="https://user-images.githubusercontent.com/63437136/173267841-916d27df-2727-4259-b8ab-a60973bd7362.png">

**Comment by zhang73331** [comment]
等下试试看一下

**Comment by dangpeng1** [comment]
成功了谢谢兄弟

**Comment by wabcwell** [comment]
成功了,,,立竿见影

**Comment by sarabear** [comment]
请问下这个luci-proto-ipv6的ipk哪里可以下到？  自带的软件包里面没这个。。。

**Reply by sarabear** [reply]
好了，要升到6.11的版本才有这个，6.10的都没，因为软件源地址也有更新了

**Comment by AshakurasA** [comment]
之前IPv6是有了，也能访问，但是IPv6 WAN 状态 不显示数据，强迫症犯了，安装这个IPK后首页终于有数据了

**Comment by wabcwell** [comment]
不知道是不是幻觉 ，，，安装完luci-proto-ipv6 后 特别容易断流

**Reply by smomop** [reply]
检查是不是其他配置的问题，ipv6本身配置就比较麻烦的
